runtime/doc/pi_netrw.txt

```diff
@@ -1510,6 +1510,9 @@ Netrw determines which special handler by the following method:
     If g:netrw_browsex_viewer == '-', then netrwFileHandlers#Invoke() will be
     used instead (see |netrw_filehandler|).
 
+    If the viewer doesn't support to handle a remote URL directry, set
```

Comment:
directry → directory

I'm not a native speaker, but grammar also does not look right. I think it should be:
```
If the viewer doesn't support handling of a remote URL directory, set …
```

Reply:
direclty was a good term here

Reply:
I agree with @dpelle about the grammar, but suspect that @eirnym is correct (despite the spelling mistake 😄 ) in that the word we're after is "directly".

```
If the viewer doesn't support handling of a remote URL directly, set …
```

Reply:
I meant "directly", but it can be removed.
Updated based on @nickspoons' suggestion.

Reply:
@nickspoons thanks, for a pointing me to a grammar mistake. Fast reading autocorrects grammar :)